mtools/mloginfo/sections/connection_section.py

```diff
@@ -37,10 +39,28 @@ def run(self):
 
         ip_opened = defaultdict(lambda: 0)
         ip_closed = defaultdict(lambda: 0)
+        
         socket_exceptions = 0
 
+        genstats = self.mloginfo.args['stats']
+        if genstats:
+            genstats = True
```

Comment:
Not needed (`genstats` is already True per the previous line)

Reply:
I was earlier using `self.mloginfo.args['stats']` directly in comparison, now  `genstats = True` is redundant. Removed.